build.sbt

```diff
@@ -275,6 +269,12 @@ lazy val testServer = (projectMatrix in file("testing/server"))
 
 lazy val testServer2_13 = testServer.jvm(scala2_13)
 
+val scalajava8compactVersion: Option[(Long, Long)] => String = {
```

Comment:
`compat`, not `compact`. But do we need this as we are targeting java 11?

Reply:
Needed by scala 2_11 code

Reply:
I will have to created separate version of class only for 2.11, does not seem to be worth it

Reply:
what is the problematic class?

Reply:
classes for httpclient in core
![image](https://user-images.githubusercontent.com/59844835/165326343-390c6533-8e23-459c-90b1-ba9c6f168d2a.png)

Reply:
yeah ... well, I guess we've got to keep it unfortunately ;) until we drop 2.11 support at least